### DISCUSSION TITLE
Colours

### DISCUSSION THREAD

**Discussion by bameyrick** [ONSdigital/design-system#1394] [Feedback]
Use this place to discuss [colours](https://ons-design-system.netlify.com/syltes/colours) in the [ONS Design System](https://ons-design-system.netlify.com)

**Comment by bameyrick** [comment]
After speaking with @will-madebymoon we noticed that the secondary button background colour should be `#dedede`, and the text colour should be `#222222` with regular font weight.

**Comment by jrbarnes9** [comment]
Focus colour is out of date. And highlight should use the same shade of yellow as the new focus colour.

**Comment by JNATION85** [comment]
$color-amber lightness to make white on amber colour combination accessible

**Comment by jrbarnes9** [comment]
`$color-grey-4` missing from colours documentation

**Comment by jrbarnes9** [comment]
`$color-grey-4` is too close to `$color-grey-3` in level of lightness.

![Screenshot 2020-09-23 at 15 34 48](https://user-images.githubusercontent.com/43346934/94027102-65ac0600-fdb2-11ea-9cc3-78764d5d8558.png)

## Suggestion 
Darken `$color-grey-3` to: `#acacac`

Currently color-grey-3 is used for:
- collapsible and text-indent content border-left (change to use $color-grey-4 instead)
- download resources dividing borders (change to use $color-grey-4 instead)
- download image border (change to use $color-grey-4 instead)
- sortable table th arrows (keep new color-grey-3)
- patternlib table th and td boder-bottom (keep new color-grey-3)
 

We can then use the darker `$color-grey-3` for `footer__hr` (it currently uses `$color-grey-2` which is too dark see: #1026)

![Screenshot 2020-09-23 at 16 56 02](https://user-images.githubusercontent.com/43346934/94037843-aeb58780-fdbd-11ea-8de7-08ac5254bec0.png)

**Comment by jrbarnes9** [comment]
Wrong colour for `btn-secondary` doucmented

Should be `#d0d0d0` ($color-button-secondary: $color-grey-4), not `#f0f0f0` ($color-grey-5)

Needs updating in the accessiblity colour combination too

**Comment by armstrongb** [comment]
It would be good to meet and discuss how these will align with the central colour palette as the work Leah and @JNATION85 did on the harmonised palette should be incorporated.

**Comment by jrbarnes9** [comment]
The following documented colour combination doesn't pass WCAG AA, as stated.  

I'm not sure it's used anywhere in the DS any more, so could probably be removed or updated with the combination: FG #4263C2 on BG #F0F0F0, as used in the [featured article](https://ons-design-system.netlify.app/components/articles/examples/articles-featured/) component

<img width="299" alt="Screenshot 2020-10-22 at 16 42 50" src="https://user-images.githubusercontent.com/43346934/96896731-c5ff9780-1485-11eb-864f-493f0e6a6568.png">

**Comment by jrbarnes9** [comment]
Broken anchor link in focus colour combination documentation
<img width="960" alt="Screenshot 2021-03-26 at 15 47 05" src="https://user-images.githubusercontent.com/43346934/112657662-8d0b8a80-8e4a-11eb-84e0-a78d67763518.png">

**Comment by kali-e** [comment]
On [this page](https://service-manual.ons.gov.uk/design-system/foundations/colours#colour-usage), the colour variable for Error - Primary Border says `--color-errors`, should this be `--ons-color-errors`? Similar for **Information - Status Indicator**, **Success - Status Indicator**, **Pending - Status Indicator** and **Success - Panel Border** on the service manual same page.

**Reply by MagdalenaLarge** [reply]
Hi @kali-e ,
Thanks for reaching out! We’ll take a look at the issue you mentioned very soon.
